nx-X11/config/cf/linux.cf

```diff
@@ -733,6 +733,21 @@ XCOMM binutils:	(LinuxBinUtilsMajorVersion)
 # define ServerExtraDefines	-DGCCUSESGAS XFree86ServerDefines
 #endif
 
+#ifdef MipselArchitecture
+# ifndef OptimizedCDebugFlags
+#  define OptimizedCDebugFlags	DefaultGcc2MipsOpt
+# endif
+# define LinuxMachineDefines	-D__MIPSEL__
+# define ServerOSDefines	XFree86ServerOSDefines -DDDXTIME
+# ifdef Mips64elArchitecture
```

Comment:
> Did you take the AsOutputArchSize stuff from the SparcArchitecture block? It looks like we haven't ever used AsOutputArchSize in the first place, so may as well not include it in the Mipsel block...

You are right dropping AsOutputArchSize here, as it seems unused elsewhere. Also removing it from the Sparc part, as that was indeed the place I imitated.